json/src/extractor.nr

```diff
@@ -73,41 +73,16 @@ pub fn extract(
         // Update state based on current byte
         let update_output = state_update_hasher(update_input);
 
-        // Create rewrite stack input
-        let rewrite_input = RewriteStackInput {
-            stack: update_output.next_stack,
-            tree_hash: update_output.next_tree_hash,
-            read_write_value: Field::from(1),
-            read_start_brace: is_equal(data[data_idx], START_BRACE),
-            read_start_bracket: is_equal(data[data_idx], START_BRACKET),
-            read_end_brace: is_equal(data[data_idx], END_BRACE),
-            read_end_bracket: is_equal(data[data_idx], END_BRACKET),
-            read_colon: is_equal(data[data_idx], COLON),
-            read_comma: is_equal(data[data_idx], COMMA),
-            read_quote: is_equal(data[data_idx], QUOTE),
-            escaped: update_input.escaped,
-            parsing_primitive: update_input.parsing_primitive,
-            parsing_string: update_input.parsing_string,
-            next_parsing_string: update_output.next_parsing_string,
-            next_parsing_primitive: update_output.next_parsing_primitive,
-            byte: data[data_idx],
-            polynomial_input: ciphertext_digest,
-            monomial: update_input.monomial,
-        };
-
-        // Rewrite stack based on current state
-        let rewrite_output = rewrite_stack(rewrite_input);
```

Comment:
`state_update_hasher` also runs `rewrite_stack`, so I just used the output from the same circuit like circom.
https://github.com/pluto/web-prover-circuits/blob/c0b31bf925fb8a0bec939e8b8167d1a8f8ad2703/circuits/json/extraction.circom#L58-L63